roles/libvirt_manager/tasks/create_vms.yml

```diff
@@ -4,15 +4,23 @@
   vars:
     _base_img_name: >-
       {{
-        (vm_data.value.image_local_dir | default(ansible_user_dir),
-         vm_data.value.disk_file_name) |
-         path_join
+        (
+          vm_data.value.image_local_dir | default(ansible_user_dir),
+          vm_data.value.disk_file_name
+        ) | ansible.builtin.path_join
       }}
     _img: >-
```

Comment:
i.e. here we're creating the overlay. Just add a task after that to inject the IP configuration? Else, the overlay is mostly useless if you copy N times the base image...

Reply:
Thanks @cjeanner for the suggestion. It was works flawlessly in the new approach.

Done. The latest commit modifies the overlay instead.